### PR TITLE
Feature/default bins

### PR DESCRIPTION
**Pull request by cristianmatache**
- alpha-build 0.1.0
- renamed build-support/make/core/ to build-support/alpha-build/core
- added default values for prettier and markdownlint binaries
- simplified pytest

### CODE REVIEW

File: build-support/alpha-build/core/markdown/format.mk [hunk 1]
```diff
@@ -2,8 +2,12 @@
 # This file is part of AlphaBuild core, don't edit it in a repo other than https://github.com/cristianmatache/alpha-build/
 # Please submit an issue/pull request to the main repo if you need any changes in the core infrastructure.
 # Before doing that, you may wish to consider:
-# - updating the config files in build-support/make/config/ to configure tools for your own use case
-# - writing a new custom rule, in build-support/make/extensions/<lang>/ and import it in the main Makefile
+# - updating the config files in build-support/alpha-build/config/ to configure tools for your own use case
+# - writing a new custom rule, in build-support/alpha-build/extensions/<lang>/ and import it in the main Makefile
+
+ifndef MARKDOWNLINT_BIN
```
Comment (PereteanuGeorge) [inline build-support/alpha-build/core/markdown/format.mk:8]: Should we put ifndef for prettier as well?

File: 3rdparty/py-env/dev-requirements.txt [hunk 2]
```diff
@@ -11,6 +11,7 @@ nbstripout
 flynt
 pyupgrade
 shfmt-py
+rstfmt
```
Comment (PereteanuGeorge) [inline 3rdparty/py-env/dev-requirements.txt:14]: Should we remove this since it is not used anymore?

Comment (cristianmatache) [reply 3rdparty/py-env/dev-requirements.txt]: good point, thank you

File: build-support/alpha-build/core/markdown/format.mk (hunk 1 above)
Comment (cristianmatache) [reply build-support/alpha-build/core/markdown/format.mk:8]: yes, thanks

**Review by PereteanuGeorge — APPROVED** [review]

**Pull request merged**